### PR TITLE
test: e2e test

### PR DESCRIPTION
**Pull request by matt-user**
closes FRO-141

### CODE REVIEW

File: packages/app/playwright/e2e/Bridge.test.ts [hunk 1]
```diff
@@ -1,19 +1,170 @@
...
+    await depositButton.click();
+    await metamask.confirmTransaction();
+
+    const postDepositBalance = await client.getBalance({
+      address: account.address,
+    });
+
+    expect(
+      ((prevDepositBalance - postDepositBalance) / BigInt(1e18)).toString()
+    ).toBe('1');
```
Comment (LuizAsFight) [inline packages/app/playwright/e2e/Bridge.test.ts:82]: can we use `bn` to do this type of checking ?

File: packages/app/playwright/e2e/Bridge.test.ts [hunk 2]
```diff
@@ -1,19 +1,170 @@
...
+    await bridgePage.first().click();
+
+    // Go to the withdraw page
+    const withdrawPage = getButtonByText(page, 'Withdraw from Fuel');
+    await withdrawPage.click();
+
+    const preWithdrawBalanceFuel = await fuelWallet.getBalance(NativeAssetId);
+    const prevWithdrawBalance = await client.getBalance({
+      address: account.address,
+    });
```
Comment (LuizAsFight) [inline packages/app/playwright/e2e/Bridge.test.ts:123]:
```suggestion
    const preWithdrawBalanceFuel = await fuelWallet.getBalance(NativeAssetId);
    const preWithdrawBalanceEth = await client.getBalance({
      address: account.address,
    });
```

File: packages/app/playwright/e2e/fixtures.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    const zipFileStream = fs.createWriteStream(zipFile);
+    // TODO fetch the exact version of wallet to avoid breaking ci
+    const zipPromise = new Promise((resolve, reject) => {
+      https
+        .get(extensionUrl, (res) => {
+          res.pipe(zipFileStream);
+          // after download completed close filestream
+          zipFileStream.on('finish', async () => {
+            zipFileStream.close();
+            console.log('Download Completed extracting zip...');
```
Comment (LuizAsFight) [inline packages/app/playwright/e2e/fixtures.ts:34]: skip comment  lint for this file

File: packages/app/playwright/e2e/fixtures.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    // setup metamask
+    // TODO sometimes this step is flaky, but I'm not sure how to fix
+    await initialSetup(chromium, {
+      secretWordsOrPrivateKey:
+        'test test test test test test test test test test test junk',
+      rpcUrl: 'http://localhost:8080',
+      network: 'localhost',
+      password: 'Tester@1234',
+      enableAdvancedSettings: true,
+    });
```
Comment (LuizAsFight) [inline packages/app/playwright/e2e/fixtures.ts:77]: can we put this key/password in a mock file to reuse if need?

File: packages/app/src/systems/Chains/eth/containers/TxEthToFuelDialog.tsx [hunk 5]
```diff
@@ -22,7 +22,7 @@ export function TxEthToFuelDialog() {
           <Text color="intentsBase12" fontSize="sm">
             Transaction: {shortAddress(metadata.txId)}
           </Text>
-          <Dialog.Close />
+          <Dialog.Close aria-label="Close unlock window" />
```
Comment (LuizAsFight) [inline packages/app/src/systems/Chains/eth/containers/TxEthToFuelDialog.tsx:25]:
```suggestion
          <Dialog.Close aria-label="Close Transaction Dialog" />
```

File: packages/app/src/systems/Chains/fuel/containers/TxFuelToEthDialog.tsx [hunk 6]
```diff
@@ -21,7 +21,7 @@ export function TxFuelToEthDialog() {
           <Text fontSize="sm" color="intentsBase12">
             Transaction: {shortAddress(metadata.txId)}
           </Text>
-          <Dialog.Close />
+          <Dialog.Close aria-label="Close unlock window" />
```
Comment (LuizAsFight) [inline packages/app/src/systems/Chains/fuel/containers/TxFuelToEthDialog.tsx:24]:
```suggestion
          <Dialog.Close aria-label="Close Transaction Dialog" />
```

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/app/playwright/commons/wallet.ts [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  // Enter password
+  const enterPassword = signupPage.locator(`[aria-label="Your Password"]`);
+  await enterPassword.type(WALLET_PASSWORD);
+  // Confirm password
+  const confirmPassword = signupPage.locator(`[aria-label="Confirm Password"]`);
+  await confirmPassword.type(WALLET_PASSWORD);
+  const toFinish = getButtonByText(signupPage, 'Next: Finish set-up');
+  await toFinish.click();
+
+  await signupPage.waitForTimeout(9000);
```
[the remark this answers is not in the document]
Comment (matt-user) [reply packages/app/playwright/commons/wallet.ts]: I wasn't able to get the tests to work with `waitForTimeout` :/

**Comment by matt-user** [comment]
> would be also nice to check steps component progression during the test. checking if icon done appeared, if correct step is selected, etc

I'm not sure if it is possible to check if the icon changes.  All I think we can check is if the text changes.

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
when running locally it fails here : 

image

in the second transaction of metamask (withdraw), gets locked here \/, test would need to click in "I want to proceed anyway"
image

File: package.json [hunk 8]
```diff
@@ -32,6 +32,7 @@
     "node:clean": "make -C ./docker clean",
     "prettier:check": "prettier --check .",
     "prettier:format": "prettier --write .",
+    "test:all": "run-s test test:e2e",
```
Comment (LuizAsFight) [inline package.json:35]: can we do `run-p` ?

**Review by LuizAsFight — APPROVED** [review]
congratz on this work @matt-user

**Pull request merged**